.gitignore

```diff
@@ -1,7 +1,6 @@
 pkg/
 Gemfile.lock
 Gemfile.local
-vendor/
```

Comment:
Gotcha. I was more trying to say that having vendor in .gitignore impacted us when we tried to use the recommended /build/vendor/custom, so I didn't think it would impact that process to remove it completely.